src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Nullable.cs

```diff
@@ -383,8 +445,34 @@ static void G(A a, B b)
     }
 }";
             var comp2 = CreateCompilation(source2, parseOptions: TestOptions.Regular8, references: new[] { comp.EmitToImageReference() });
-            // PROTOTYPE(NullableReferenceTypes): Should report warnings.
-            comp2.VerifyEmitDiagnostics();
+            comp2.VerifyDiagnostics(
+                // (8,14): warning CS8620: Nullability of reference types in argument of type 'A' doesn't match target type 'I<(object, object?)>' for parameter 'b' in 'void C.F(I<(object, object)> a, I<(object, object?)> b)'.
+                //         F(a, a);
+                Diagnostic(ErrorCode.WRN_NullabilityMismatchInArgument, "a").WithArguments("A", "I<(object, object?)>", "b", "void C.F(I<(object, object)> a, I<(object, object?)> b)").WithLocation(8, 14),
+                // (9,11): warning CS8620: Nullability of reference types in argument of type 'B' doesn't match target type 'I<(object, object)>' for parameter 'a' in 'void C.F(I<(object, object)> a, I<(object, object?)> b)'.
+                //         F(b, b);
+                Diagnostic(ErrorCode.WRN_NullabilityMismatchInArgument, "b").WithArguments("B", "I<(object, object)>", "a", "void C.F(I<(object, object)> a, I<(object, object?)> b)").WithLocation(9, 11));
+        }
+
+        [Fact]
+        public void EmitAttribute_Constraint()
```

Comment:
>EmitAttribute_Constraint [](start = 20, length = 24)

I'm not sure the best way to track this, but I would suggest leaving some PROTOTYPE markers on tests that will need to be duplicated once a flag is implemented for turning the feature on/off.
The tests modified in this PR would be good candidates.